src/Middleware/WebSockets/test/UnitTests/WebSocketMiddlewareTests.cs

```diff
@@ -85,6 +85,7 @@ public async Task SendEmptyData_Success()
         }
 
         [Fact]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/32297")]
```

Comment:
Should probably just quarantine the whole class or wait until I look into these tests later today. The tests weren't really running for the last few years and I recently fixed that, but looks like there is the same race with most of them.

Reply:
We have a way of quarantining a whole class?

Reply:
Yeah, the attribute can be applied to the class